config/routes.rb

```diff
@@ -328,6 +328,10 @@ def scoped_navigation_routes(context, navigation)
           resources :documents, only: [:show]
         end
 
+        devise_for :users, path: "hub", controllers: {
```

Comment:
You're right that users are in the hub (and you can even see that this is creating a namespaced hub path within the ctc area, which is also somewhat unexpected) and we're trying to logout clients, so i think we'll want to mount `devise_for :clients` instead.

Line 122 of the routes file shows how we're mounting the devise engine for gyr login/logout routes.

We might need to look at the logout logic in general -- it's possible that across the app it's configured through the routes for user which would be quite odd to me. 

Although it looks like line 122 specfiically routes everything but the sessions functionality, so potentially this is by design. Do you know anything about that @alaroia-cfa